reference/camera.md

```diff
@@ -520,3 +527,64 @@ images, the `quality` parameter is ignored.
 The return value of the `wb_camera_save_image()` is 0 in case of success. It is
 -1 in case of failure (unable to open the specified file or unrecognized image
 file extension).
+
+---
+
+**Name**
+
+**wb\_camera\_has\_recognition**,**wb\_camera\_recognition\_enable**,**wb\_camera\_recognition\_disable**,**wb\_camera\_recognition\_get\_sampling\_period**,**wb\_camera\_recognition\_get\_number\_of\_objects**,**wb\_camera\_recognition\_get\_objects**  - *camera recognition functions*
+
+{[C++](cpp-api.md#cpp_camera)}, {[Java](java-api.md#java_camera)}, {[Python](python-api.md#python_camera)}, {[Matlab](matlab-api.md#matlab_camera)}, {[ROS](ros-api.md)}
+
+```c
+#include <webots/camera.h>
+
+bool wb_camera_has_recognition(WbDeviceTag tag);
+void wb_camera_recognition_enable(WbDeviceTag tag, int sampling_period);
+void wb_camera_recognition_disable(WbDeviceTag tag);
+int wb_camera_recognition_get_sampling_period(WbDeviceTag tag);
+int wb_camera_recognition_get_number_of_objects(WbDeviceTag tag);
+const WbCameraRecognitionObject *wb_camera_recognition_get_objects(WbDeviceTag tag);
+```
+
+**Description**
+
+If a [Recognition](recognition.md) node is present in the `recognition` field, the camera can recognize objects in its image.
+
+The `wb_camera_has_recognition` function can be used to determine whether a [Recognition](recognition.md) node is present or not.
+
+The `wb_camera_recognition_enable()` function allows the user to enable the recognition, note that it is not needed to enable the camera to let recognition work.
```

Comment:
You can also remove the "the" before "recognition".  

"enable recognition, note that it is not needed to enable the camera to let recognition work." -> "..enable recognition. It is not necessary to enable the camera for recognition to work."